index.html

```diff
@@ -21,20 +25,23 @@
         <nav class="navbar">
           <a href="#" class="logoo">Tuzy Orizu</a>
           <div class="place hide">
+            <span class="close"><img src="img/Icons/cancel.png" alt="" /></span>
             <ul class="nav-menu">
               <li class="nav-item">
-                <a href="#" class="nav-link">Portfolio</a>
+                <a href="#portfolio" class="nav-link">Portfolio</a>
               </li>
               <li class="nav-item">
-                <a href="#" class="nav-link">About</a>
+                <a href="#about" class="nav-link">About</a>
               </li>
               <li class="nav-item">
-                <a href="#" class="nav-link">Contact</a>
+                <a href="#contact" class="nav-link">Contact</a>
               </li>
             </ul>
           </div>
           <div class="hamburger">
-            <img src="img/Icons/Union.png" alt="hamburger-menue" />
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
```

Comment:
- [ ] As requested by the previous reviewer, Please make sure the design matches the Figma design as much as possible. And fix the below issues.
  - Please use the correct color and size for the close(X) button.
  - Please use the correct font-weight for the mobile menu options.
  - Please add the blur effect in the background (You can use this CSS property `backdrop-filter: blur(5px);`)

### Your design

![image](https://user-images.githubusercontent.com/83803414/167471961-d5c8958a-903f-4a8d-beca-1af0fea9146c.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/167471983-2783a85d-3753-4a32-9cfc-4d6cb748110d.png)